pkg/block/block.go

```diff
@@ -103,10 +102,6 @@ func Upload(ctx context.Context, logger log.Logger, bkt objstore.Bucket, bdir st
 		return errors.Wrap(err, "encode meta file")
 	}
 
-	if err := bkt.Upload(ctx, path.Join(DebugMetas, fmt.Sprintf("%s.json", id)), strings.NewReader(metaEncoded.String())); err != nil {
```

Comment:
Hm, decision was to remove this.. optionally? 

TBH I never used, but wonder if this was not used by someone at the end.

Reply:
One way would be to remove .. let's say by default but enable for opt-in? 🤔

Reply:
I'm fine to fully remove it. I know @pstibrany used it few times, so let me ask it too.

Reply:
I can only repeat what I wrote here: https://github.com/thanos-io/thanos/issues/3839#issuecomment-786674853

I don’t have strong feelings about it, but found it useful in the past.

Reply:
Me too, that's why I created this in first place